shanoir-ng-studies/src/main/java/org/shanoir/ng/center/service/CenterServiceImpl.java

```diff
@@ -82,9 +96,49 @@ protected Center updateValues(final Center from, final Center to) {
 		to.setWebsite(from.getWebsite());
 		return to;
 	}
+	
+	@Override
+	public Center update(Center center) throws EntityNotFoundException {		
+		final Center centerDb = centerRepository.findOne(center.getId());
+		if (centerDb == null) {
+			throw new EntityNotFoundException(center.getClass(), center.getId());
+		}
+		String previousName = centerDb.getName();
+		updateValues(center, centerDb);
+		Center updatedCenter = centerRepository.save(centerDb);
+		// send name update via rabbitmq
+		if (!previousName.equals(updatedCenter.getName())) {
+			try {
+				updateName(new IdName(center.getId(), center.getName()));
+			} catch (MicroServiceCommunicationException e) {
+				LOG.error("Could not send the center name change to the other microservices !", e);
+			}
+		}
+		return updatedCenter;
+	}
+	
+	@Override
+	public Center create(Center center) {
+		try {
+			updateName(new IdName(center.getId(), center.getName()));
+		} catch (MicroServiceCommunicationException e) {
+			LOG.error("Could not send the center name creation to the other microservices !", e);
+		}
+		return super.create(center);
+	}
 
 	@Override
 	public Center findByName(String name) {
 		return centerRepository.findByName(name);
 	}
+	
+	private boolean updateName(IdName idName) throws MicroServiceCommunicationException{
+		try {
+			rabbitTemplate.convertAndSend(RabbitMQConfiguration.centerNameUpdateQueue().getName(),
```

Comment:
Can you use the constant CENTER_NAME_UPDATE_QUEUE here please ? It's how it's done everywhere else and I find it more usefull when looking for queue's usage